app/controllers/hubstats/deploys_controller.rb

```diff
@@ -81,6 +80,8 @@ def create
         @deploy.pull_requests = Hubstats::PullRequest.where(repo_id: @deploy.repo_id)
                                                      .where(number: @pull_request_id_array)
 
+        @deploy.user_id = @deploy.pull_requests.first.merged_by
```

Comment:
Did we figure out how to avoid the potential race condition here if this executes before the pull request merge event webhook is processed?